setup.sh

```diff
@@ -138,7 +138,7 @@ function install_macos() {
 # Install bot Debian_ubuntu
 function install_debian() {
     sudo apt-get update
-    sudo apt-get install -y build-essential autoconf libtool pkg-config make wget git
+    sudo apt-get install -y build-essential autoconf libtool pkg-config make wget git libpython3-dev
```

Comment:
While i'm aware of this - i'm not sure if we should be adding this back (we used to have this in the setup.sh file before).

This makes ... certain assumptions about your python installation (it will break if it's not a package manager based installation), and it's not required in all cases (depending on the availability of wheels for your python version).